cores/esp32/MacAddress6.cpp

```diff
@@ -0,0 +1,76 @@
+#include <MacAddress6.h>
+#include <stdio.h>
+
+//Default constructor, blank mac address.
+MacAddress6::MacAddress6() {
+    _mac.val = 0;
+}
+
+MacAddress6::MacAddress6(uint64_t mac) {
+    _mac.val = mac;
+}
+
+MacAddress6::MacAddress6(const uint8_t *macbytearray) {
+    memcpy(_mac.bytes, macbytearray, sizeof(_mac.bytes));
+}
+

```

Comment:
Maybe adding MacAddress6::MacAddress6(uint8_t b1, uint8_t b2, uint8_t b3, uint8_t b4, uint8_t b5, uint8_t b6) constructor?

Reply:
I considered this but couldn't think of a valid use case.  For that many bytes I would handle it as an array.  No use cluttering up the class if it doesn't add meaningful functionality.

Reply:
I see. You are thining as a developer... we have also the "Arduiners" - they think it in simpler way. :-)